.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
```

Comment:
we dont want to overcomplicate this here. people should not be forced to run vanilla.
yes that CAN influence stuff, but the probability as very low. as low as the amount of people who will not be confused by this request.